code/modules/client/preference_setup/loadout/loadout_xeno/human.dm

```diff
@@ -87,3 +88,13 @@
 	path = /obj/item/clothing/accessory/badge/passcard/scarab
 	whitelisted = list(SPECIES_HUMAN_OFFWORLD)
 	sort_category = "Xenowear - Human"
+	culture_restriction = list(/singleton/origin_item/origin/coa_spacer)
+
+/datum/gear/accessory/offworlder_mantle
+	display_name = "native burzsian mantle"
+	path = /obj/item/clothing/accessory/poncho/burzsian_mantle/native
+	whitelisted = list(SPECIES_HUMAN_OFFWORLD)
+	sort_category = "Xenowear - Human"
+	flags = GEAR_HAS_DESC_SELECTION
+	culture_restriction = list(/singleton/origin_item/origin/coa_spacer)
```

Comment:
not needed unless the specific origin is intended. (its already whitelisted to offworlders.)

Reply:
The restriction is intended, yes, because not all offworlders are from the Coalition. The Coalition Offworlders origin is the only way you can be a Scarab or native Burzsian specifically.

Reply:
👍